auslib/blobs/base.py

```diff
@@ -123,3 +124,40 @@ def getFooterXML(self, updateQuery, update_type, whitelistedDomains, specialForc
 
     def getInnerXML(self, updateQuery, update_type, whitelistedDomains, specialForceHosts):
         raise NotImplementedError()
+
+    def containsForbiddenDomain(self, product):
+        """Returns True if the blob contains any file URLs that contain a
+           domain that we're not allowed to serve updates to."""
+        # Check the top level URLs, if the exist.
+        f = open('testing', 'a')
+        f.write(str(app.config.get("WHITELISTED_DOMAINS")))
+        f.close()
+        for c in self.get('fileUrls', {}).values():
```

Comment:
This is an validator which works app releases only. Could we keep bug 1165164 and bug 1293352 in mind and put it in apprelease.py instead ? We could subclass, and just make the GMP/SystemAddon copies no-ops for now.